HelpSource/Guides/Tour-of-Special-Functions.schelp

```diff
@@ -339,26 +344,26 @@ CODE::
 (
 var erfSamps = (-3, -2.995 .. 3);
 var plotSpecs = [
-    [   \erf,
-        erfSamps,
-        "erf(z)\nError Function",
-        -1, 1
-    ],
-    [   \erfC,
-        erfSamps,
-        "erfC(z)\nComplement of the Error Function",
-        0, 2
-    ],
-    [   \erfInv,
-        (-0.9995, -0.995 .. 1),
-        "erfInv(z)\nInverse of the Error Function",
-        -3, 2
-    ],
-    [   \erfCInv,
-        (0.001, 0.002 .. 1.999),
-        "erfCInv(z)\nInverse of the Complement of the Error Function",
-        -3, 2
-    ]
+	[   \erf,
+		erfSamps,
+		"erf(z)\nError Function",
+		-1, 1
+	],
+	[   \erfC,
+		erfSamps,
+		"erfC(z)\nComplement of the Error Function",
+		0, 2
+	],
+	[   \erfInv,
+		(-0.9995, -0.995 .. 1),
+		"erfInv(z)\nInverse of the Error Function",
+		-3, 2
+	],
+	[   \erfCInv,
+		(0.001, 0.002 .. 1.999),
+		"erfCInv(z)\nInverse of the Complement of the Error Function",
+		-3, 2
+	]
```

Comment:
The indentation should be done in groups of 4 spaces everywhere ideally.

Reply:
I agree! I'd love it if all sc, schelp, and c++ used spaces. Unfortunately the current rule is tabs.
https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#indentation-1
Happy to submit a future PR un-tabifying everything if the rule is changed.

Reply:
You're right. The current rule is tabs. Hopefully, style guidelines will change in 2019 :)

Reply:
it's tabs for sc code, and i'm pretty sure it's going to stay tabs.  why would you want it to change? this has already been discussed at length btw on ths spacing issue. (#2819)

Reply:
Ah! We are only spacifying C/C++ project files. Good to know.